projects/ad4630_fmc/zed/README.md

```diff
@@ -1,36 +1,114 @@
+# EVAL-AD463X-FMC/ZED HDL Project
 
-# EVAL-AD463X_FMCZ HDL reference design
+## Building the project
 
-## Building the design
+The parameters configurable through the `make` command, can be found below, as
+well as in the **system_project.tcl** file.
 
-The design supports almost all the digital interface modes of AD463x, AD403x 
-and adaq42xx a new bit stream should be generated each time when the targeted 
+```
+cd projects/ad4630_fmc/zed
+make
+```
+
+This design supports almost all the digital interface modes of AD463x, AD403x
+and ADAQ42xx. A new bit stream should be generated each time when the targeted
 configuration changes.
 
-Default configuration: generic SPI mode for clocking, 2 lanes per channel, SDR 
-data capture and capture zone 2. 
+If other configurations are desired, then the parameters from the HDL project
+need to be changed, as well as the Linux project configurations:
+
+The overwritable parameters from the environment are:
```

Comment:
Please write ``(default)`` to each parameter to know the default value from the beginning, and not to check the example configurations to know it